### DISCUSSION TITLE
Upload mode.

### DISCUSSION THREAD

**Discussion by royrajan95** [kalanakt/All-Url-Uploader#85] [General]
Hi, I deployed in Local VPS

![format](https://user-images.githubusercontent.com/84740586/202506571-7781aee8-eaa5-4d24-b1d5-6541935e3ba7.JPG)

as shown in the image while sending links to the bot, it replies with 2 options.
SMEDIA & VIDEO
But both the buttons give the same result as a video file.

Can we add Upload as "SVideo / Document File" option as in the below image??

![format 1](https://user-images.githubusercontent.com/84740586/202508676-deaa8cc1-0a0c-47af-8a8a-0b700e8a231e.JPG)

**Comment by kalanakt** [comment]
I'll add that 🚀

**Reply by royrajan95** [reply]
thanks.
